tests/integration/test_docker.py

```diff
@@ -52,6 +52,9 @@ def test_docker():
     assert "172" in cont1.get_ip()
     assert "sbin" in cont1.execute("ls /")
     cont1.install_packages("nc")
+    # Check if files are present in container
+    assert cont1.check_file_exists("/usr/bin/ncat")
+    assert_raises(subprocess.CalledProcessError, cont1.check_file_exists, "/bin/bin")
```

Comment:
I think this interface is horrible:
1. the function should return False if the file is not present, yet it raises an exception
2. the exception is not documented
3. the exception does not come from this library